src/os/open.zig

```diff
@@ -2,14 +2,10 @@ const std = @import("std");
 const builtin = @import("builtin");
 const Allocator = std.mem.Allocator;
 
-const log = std.log.scoped(.@"os-open");
+const apprt = @import("../apprt.zig");
+const CircBuf = @import("../datastruct/circ_buf.zig").CircBuf;
```

Comment:
On a future note, your PRs have a pattern of containing unrelated changes such as this. I'm going to revert all the circular buffer stuff. It makes sense but conflating the purpose and goal of a PR makes review harder and makes things like reverting uglier if we ever to. I'd appreciate keeping things tightly scoped.